package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "arc_hive",
+  "version": "1.0.0",
```

Comment:
May want to set this to 0.0.0 for now - versions are subjective/convention, but often 1.0 means first stable complete release.